src/structures/ClientApplication.js

```diff
@@ -23,6 +30,33 @@ class ClientApplication extends Application {
   _patch(data) {
     super._patch(data);
 
+    if ('tags' in data) {
+      /**
+       * The tags this application has (max of 5)
+       * @type {?string[]}
+       */
+      this.tags = data.tags;
+    }
```

Comment:
I'd just make this always present instead... Thoughts @discordjs/the-big-4 
```js
this.tags = data.tags ?? this.tags ?? []
```
or similar

Reply:
I'd just stick with this pattern since it's used in many places, although we can argue that we also set defaults (specially `null`) in many other places.